test/kube_testing/k8s.go

```diff
@@ -329,7 +328,7 @@ func (o *K8s) CleanupCRDs() {
 
 func (l *K8s) Cleanup() {
 	for _, result := range l.pods {
-		err := l.deletePods(result)
+		err := l.deletePods(0, result)
```

Comment:
What is timeout 0 here?

Reply:
Will update parameter name, it is gracefull timeout, to not wait for pod termination logic.